docs/html/_sources/account/th_wht.rst.txt

```diff
@@ -0,0 +1,216 @@
+
+
+.. nextslide::
+
+การตั้งค่า / สร้างเอกสารภาษีหัก ณ ที่จ่าย
+-------------
+
+ให้ติดตั้ง module Withholding Tax และ Withholding Tax Cert จาก source code ด้านบน
+เมื่อติดตั้งเสร็จสิ้น ระบบจะสามารถตั้งค่าเลือก Account ที่ต้องการให้เป็น Withholding Tax ได้ เผื่อกรณีที่ต้องการให้ระบบหักภาษี ณ ที่จ่ายแบบอัตโนมัติ
+โดยมีวิธีดังนี้ (Optional)
+
+1. ไปที่เมนู Invoicing > Configuration > Accounting > Chart of Accounts
+2. เลือก Account ที่ต้องการให้ทำเป็น Withholding Tax > ติ๊กถูกที่ Field WT Account
+
+.. image:: images/wht_coa.png
+
+
+3. ไปที่เมนู Invoicing > Configuration > Invoicing > Withholding Tax
+4. สร้างเอกสารใหม่ แล้วกำหนดค่าต่าง ๆ ที่ต้องการหักภาษี > Save
+
+.. image:: images/wht_wht.png
+- Name : ชื่อที่จะแสดง
+- Amount : ร้อยละที่ต้องการหักภาษี
+- Withholding Tax Account : account ที่เป็น withholding tax เท่านั้น (กำหนดจากข้อ 2)
+
+ในกรณีที่ต้องการให้ระบบคำนวณภาษี เมื่อเลือกสินค้าที่ต้องการ สามารถตั้งค่าได้ดังนี้
+
+1. ไปที่เมนู Invoicing > Vendors > Products
+2. เลือก Product ที่ต้องการหักภาษี ณ ที่จ่าย
+3. คลิก Purchase Tab > เลือก Withholding Tax > Save
+
+
+.. note::
+    หมายเหตุ : หากไม่เลือก WT Account ระบบไม่คำนวณการหักภาษี ณ ที่จ่ายให้อัตโนมัติ แต่สามารถให้ผู้ใช้งานเป็นคนกรอกข้อมูลในส่วนนี้ได้
+
+ขั้นตอนการทำภาษีหัก ณ ที่จ่ายในระบบ
+-------------
+
+กระบวนการทำภาษีหัก ณ ที่จ่ายสามารถทำได้ที่หน้า Payment กับ Journal Entries (สร้างตรง)
+กรณีที่มีการทำภาษีหัก ณ ที่จ่ายมากกว่า 1 บรรทัดในเอกสาร 1 ใบ
+จะต้องติดตั้ง module Withholding Tax Multi เพื่อให้ระบบสามารถทำภาษีได้มากกว่า 1
+
+1. การทำภาษีหัก ณ ที่จ่ายที่หน้า Payment
+################################################################
+
+สามารถให้ระบบคำนวณให้อัตโนมัติ หรือจะคำนวณเองก็ได้ ถ้าต้องการคำนวณเองไม่ต้องทำข้อ 3 และ 4  ซึ่งขั้นตอนการคำนวณภาษีที่หน้า Payment จะมีดังนี้
+
+1. ไปที่เมนู Invoicing > Vendors > Bills
+2. สร้างเอกสารที่ทำการซื้อและมีภาษีหัก ณ ที่จ่าย
+3. คลิกที่ 3 จุดตรง Invoice Lines > แสดง WT
+
+.. image:: images/wht_vendor_bill.png
+
+4. เลือก WT ที่ต้องการจากการตั้งค่าก่อนหน้านี้ ในกรณีที่มีการตั้งค่า Withholding Tax ใน Product นั้น ระบบจะเลือก WT ให้อัตโนมัติ
+5. Save > Post > Register Payment
+6. ระบบจะคำนวณตัวเลขให้อัตโนมัติ ในกรณีที่คำนวณเองให้ใส่จำนวนหลังหักภาษีที่ช่อง Amount และส่วนต่างให้เลือก Account Withholding Tax
+
+    1. กรณีที่ Invoices มีภาษีหัก ณ ที่จ่ายตัวเดียว ระบบจะเลือก Mark invoice as fully paid
+
+    .. image:: images/wht_payment.png
+
+
+    2. กรณีที่ Invoices มีภาษีหัก ณ ที่จ่ายมากกว่า 1 และติดตั้ง module Withholding Tax Multi ระบบจะเลือก Mark invoice as fully paid
+    (multi deduct)หากต้องการคำนวณเองแบบมี Withholding Tax มากกว่า 1 สามารถทำได้โดยการติดตั้ง module `account_payment_multi_deduction <https://github.com/OCA/account-payment/tree/13.0/account_payment_multi_deduction>`_
+    แทน module Withholding Tax Multi
+
+    .. image:: images/wht_multi.png
+        :align: center
+
+    .. image:: images/wht_payment_multi.png
```

Comment:
Add some highlight should be good.
![image](https://user-images.githubusercontent.com/1973598/98921805-49039300-2504-11eb-987a-e29a08407063.png)